README.md

```diff
@@ -96,6 +96,7 @@ The following resources may also be useful:
  * [Software Emulation with Renode](tensorflow/lite/micro/docs/renode.md)
  * [Python Dev Guide](docs/python.md)
  * [Automatically Generated Files](docs/automatically_generated_files.md)
+ * [Arm IP support](tensorflow/lite/micro/docs/arm.md)
```

Comment:
Sorry, I should have comment on this before. Could you make ARM IP support a sublevel of a new level like below?

* [Platform/IP support]     --> Level 0
  * [Arm IP support](tensorflow/lite/micro/docs/arm.md)  --> Level1

New Platform support should means how to add a new platform.

Else, LGTM

Reply:
Done. I didn't add Renode as subchapter, but maybe it should be there... WDYT?

Reply:
Potentially. I am currently a little ambivalent simply because the doc is more about emulation only. I have approved your PR as it is. We can handle renode later.